test/e2e/serial/workload_placement_test.go

```diff
@@ -506,6 +511,41 @@ var _ = Describe("[serial][disruptive][scheduler] workload placement", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(schedOK).To(BeTrue(), "pod %s/%s not scheduled with expected scheduler %s", updatedPod.Namespace, updatedPod.Name, schedulerName)
 		})
+
+		It("[test_id: 47583] a deployment with a guaranteed pod with one container should be scheduled into one NUMA zone", func() {
```

Comment:
Not sure if this is the proper test_id

Reply:
it's a label so should not be a big deal. I'd just writr `test_id:47583` (no space in between) for starters